src/AST-Core/RBProgramNodeVisitor.class.st

```diff
@@ -127,6 +128,14 @@ RBProgramNodeVisitor >> visitSuperNode: aSuperNode [
 	^ self visitVariableNode: aSuperNode
 ]
 
+{ #category : #visiting }
+RBProgramNodeVisitor >> visitTemporaryDeclarationNode: aTemporaryDeclarationNode [ 
```

Comment:
Can you use this comment for the method in the trait

Reply:
done